pftp/config.go

```diff
@@ -85,5 +124,37 @@ func defaultConfig(config *config) {
 	config.TransferTimeout = 900
 	config.KeepaliveTime = 900
 	config.ProxyProtocol = false
+	config.DataChanProxy = false
+	config.DataPortRange = ""
 	config.WelcomeMsg = "FTP proxy ready"
+	config.TransferMode = "CLIENT"
+}
+
+func dataPortRangeValidation(r string) error {
+	if len(r) == 0 {
+		return nil
+	}
+
+	lastErr := fmt.Errorf("Data port range config wrong. set default(random port)")
+	portRange := strings.Split(r, "-")
+
+	if len(portRange) != 2 {
```

Comment:
You should make const value is `2`

Reply:
Is this param really needs to be const value?

It only used for judgement the port range configuration has correct length.

And in my opinion, this setting rule it will not be change.

Reply:
changed : https://github.com/pyama86/pftp/pull/32/commits/13a858f62b16662e5dc1f19933fceafb78c22e41

Reply:
ah...
I got it.
thanks!